set.mm

```diff
@@ -51040,64 +51108,98 @@ Contrast with domain (defined in ~ df-dm ).  For alternate definitions,
       HVDVEADRVQUBUCUDUEUKUFUGDEUTBGSDEUTBUQSUHUIUTAURULUMUNUO $.
   $}
 
-  $( Ordered pair membership in a restriction.  Exercise 13 of [TakeutiZaring]
-     p. 25.  (Contributed by NM, 14-Oct-2005.)  (Revised by BJ,
-     18-Feb-2022.) $)
-  opelresg $p |- ( B e. V -> ( <. A , B >. e. ( C |` D ) <->
-                    ( <. A , B >. e. C /\ A e. D ) ) ) $=
+  $( Ordered pair elementhood in a restriction.  Exercise 13 of [TakeutiZaring]
+     p. 25.  (Contributed by NM, 13-Nov-1995.)  (Revised by BJ, 18-Feb-2022.)
+     (Revised by Peter Mazsa, 24-Sep-2022.) $)
+  opelres $p |- ( C e. V ->
```

Comment:
Why is the suffix "g" removed? It was correct according to our conventions: 

>       When  an inference is converted to a theorem by eliminating an "is a set"
>        hypothesis, we sometimes suffix the closed form with "g" (for "more
>        general") as in ~ uniex vs. ~ uniexg .  In this case, the inference
>        often has no suffix "i".

According to this rule, also the former ~opelres had a correct label, although it is an inference.

Reply:
It's true that one is the inference associated with the other (except for `V` replaced by `_V`), so this would give labels xxx/xxxi, but in this special case, as @avekens points out, it is our convention to use xxxg/xxx. Maybe we should change that convention, but for the moment it is the one we used. (There are also cases where we use xxxt/xxx... maybe one day we should be a bit more consistent, but another day).

Reply:
(Resolved below https://github.com/metamath/set.mm/pull/2873#issuecomment-1272455776 )